### PR TITLE
Filtering pods not in 'Running' phase

### PR DESCRIPTION
**Pull request by hanahmily**
As the title shows, ALS analysis will filter pods whose phase is not 'Running', that will avoid invalid pods to mess IP-mapping up.

### CODE REVIEW

File: oap-server/server-receiver-plugin/envoy-metrics-receiver-plugin/src/main/java/org/apache/skywalking/oap/server/receiver/envoy/als/K8sALSServiceMeshHTTPAnalysis.java [hunk 1]
```diff
@@ -93,8 +95,13 @@ private void loadPodInfo() {
             Map<String, ServiceMetaInfo> ipMap = new HashMap<>(list.getItems().size());
             long startTime = System.nanoTime();
             for (V1Pod item : list.getItems()) {
+                if (!item.getStatus().getPhase().equals(VALID_PHASE)) {
```
Comment (wu-sheng) [inline oap-server/server-receiver-plugin/envoy-metrics-receiver-plugin/src/main/java/org/apache/skywalking/oap/server/receiver/envoy/als/K8sALSServiceMeshHTTPAnalysis.java]: Whether the status and phase are NULL? Cause NPE maybe.

**Comment by hanahmily** [comment]
Never mind. Kubernetes API always set it to a non-null value.

Hongtao Gao

Apache SkyWalking && Apache ShardingSphere
Twitter, @hanahmily

**Comment by wu-sheng** [comment]
CI failure with check-style issue.

**Comment by hanahmily** [comment]
done.

**Pull request merged**